index.html

```diff
@@ -1375,10 +1362,10 @@
             Reconnecting to a presentation
           </h4>
           <p>
-            When the <code><dfn for="PresentationRequest" data-lt=
-            "reconnect()">reconnect</dfn>()</code> method is called, the
-            <a>user agent</a> MUST run the following steps to <dfn>reconnect to
-            a presentation</dfn>:
+            When the <code><dfn for=
+            "PresentationRequest">reconnect</dfn>()</code> method is called,
```

Comment:
Nit: I feel like we should consistently drop the parenthesis after method referencesy.  We aren't including correct parameter lists, and as long as we say "method" afterwards I don't think there's any ambiguity.